src/nni_manager/training_service/remote_machine/extends/linuxCommands.ts

```diff
@@ -48,13 +48,13 @@ class LinuxCommands extends OsCommands {
     }
 
     public allowPermission(isRecursive: boolean = false, ...folders: string[]): string {
-        const folderString = folders.join("' '");
+        const folderString = folders.join(" ");
         let command;
 
         if (isRecursive) {
-            command = `chmod 777 -R '${folderString}'`;
+            command = `chmod 777 -R ${folderString}`;
```

Comment:
Why remove the `''`?

Reply:
because shell can not expand wildcard in single quotes. For example, this line can not be executed correctly:
```
chmod 777 '/tmp/nni/*'
```
This pr will change the command to
```
chmod 777 /tmp/nni/*
```

Reply:
The quote mark is required when the folder name contains place. Please add it back.